docs/testplan/pac/Port_Access_Control.md

```diff
@@ -0,0 +1,562 @@
+# Port Access Control(PAC)
+
+[TOC]
+
+## Test Plan Revision History
+
+| Rev  | Date       | Author            | Change Description           |
+| ---- | ---------- | ----------------- | ---------------------------- |
+| 1    | 18/05/2023 | Lakshminarayana D | Initial Version of test plan |
+
+## Definition/Abbreviation
+
+| **Term**   | **Meaning**                              |
+| ---------- | ---------------------------------------- |
+| VLAN       | Virtual Local Area Network               |
+| PAC        | Port Access Control                      |
+| EAPoL      | Extensible Authentication Protocol over LAN |
+| MAC        | Media Access Control                     |
+| MAB        | Mac Authentication Bypass                |
+| PO         | Port Channel                             |
+| NAS        | Network Access Switch                    |
+| DUT        | Device Under Test                        |
+| RADIUS     | Remote Authentication Dial In User service |
+| FDB        | Forwarding Database                      |
+| Supplicant | A client that attempts to access services offered by the Authenticator |
+| PAE        | Port Access Entity                       |
+
+## Introduction
+
+### Objective
+
+The main objective of this document is to cover the test cases that will be executed for Port authentication methods 802.1x and MAB. Topologies and test cases for testing the feature will be discussed as part of this document.
+
+### Scope
+
+- PAC authentication of hosts on access port
+- This functionality has been tested using the SPyTest framework. In order to emulate 802.1x and MAB clients, traffic generators like Ixia and Spirent will be used. FreeRADIUS is using for User Authentication.
+- Currently, The PAC functionality scoping on platforms AS4630-54PE, Dell N3248TE and Dell N3248PXE.
```

Comment:
Please remove platform references, it's platform independent feature

Reply:
Removed the statement.